src/script/lua_api/l_object.cpp

```diff
@@ -1104,6 +1104,44 @@ int ObjectRef::l_get_breath(lua_State *L)
 	return 1;
 }
 
+// set_player_attribute(self, attribute, value)
+int ObjectRef::l_set_player_attribute(lua_State *L)
+{
+	ObjectRef *ref = checkobject(L, 1);
+	PlayerSAO* co = getplayersao(ref);
+	if(co == NULL) {
+		return 0;
+	}
+
+	std::string attr = luaL_checkstring(L, 2);
+	std::string value = luaL_checkstring(L, 3);
+
+	if (co->getType() == ACTIVEOBJECT_TYPE_PLAYER) {
+		co->setExtraAttribute(attr, value);
+	}
+	return 1;
+}
+
+// get_player_attribute(self, attribute)
+int ObjectRef::l_get_player_attribute(lua_State *L)
+{
+	ObjectRef *ref = checkobject(L, 1);
+	PlayerSAO* co = getplayersao(ref);
+	if(co == NULL) return 0;
```

Comment:
I think @kwolekr doesnt like the return on the same line, and also space after if.